test_batteryChecker.c

```diff
@@ -0,0 +1,19 @@
+#include <assert.h>
+#include "batteryChecker.h"
+
+float DummyInputParam[6][3] = { {20,70,2},{50,60,0.7},{30,60,1},{20,30,2},{50,10,1},{30,45,6} };
```

Comment:
Test cases need to be strengthened further. At least minimum set of permutation and combinations of all three parameter values should be considered. For example one parameter will have 2 boundary values and 3 in between values, i.e. <, ==, > threshold values.
Having strong test cases is always helpful in maintaining code quality and guarding against accidental changes during extension.